navigation/navigation-common/src/androidMain/kotlin/androidx/navigation/NavDestination.android.kt

```diff
@@ -59,7 +59,7 @@ actual constructor(public actual val navigatorName: String) {
         private val hasMatchingAction: Boolean,
         private val mimeTypeMatchLevel: Int
     ) : Comparable<DeepLinkMatch> {
-        override fun compareTo(other: DeepLinkMatch): Int {
+        public actual override fun compareTo(other: DeepLinkMatch): Int {
```

Comment:
is android source set now involved in our build somehow? Or is it just an alignment with aosp?

Reply:
Alignment. It will be replaced by the next merge anyway